Config.uk

```diff
@@ -258,8 +258,57 @@ if LWIP_SOCKET
 		default y
 		help
 			Enable ppoll() implementation.
+
+	config LWIP_UDP_RECVMBOX_FACTOR
+		int "Factor for UDP recvmbox"
+		range 1 5
+		default 2
+		help
+			How many times should the UDP recvmbox be bigger than the LWIP stackthread mbox size.
+			Default 2, maximum 5.
+
+	config LWIP_TCP_RECVMBOX_FACTOR
+		int "Factor for TCP recvmbox"
+		range 1 5
+		default 2
+		help
+			How many times should the TCP recvmbox be bigger than the LWIP stackthread mbox size.
+			Default 2, maximum 5.
 endif
 
+choice
+    prompt "LWIP stackthread mbox size"
```

Comment:
I think it makes sense to place this option between `Operation mode` and `Memory allocation mode`.
I would name the prompt like: `prompt "Stack input mailbox size"`